src/seqtk/seqtk_subseq/config.vsh.yaml

```diff
@@ -0,0 +1,76 @@
+name: seqtk_subseq
+namespace: seqtk
+description: | 
+  Extract subsequences from FASTA/Q files. Takes as input a FASTA/Q file and a name.lst (sequence ids file) or a reg.bed (genomic regions file).
+keywords: [subseq, FASTA, FASTQ]
+links:
+  repository: https://github.com/lh3/seqtk/tree/v1.4
+license: MIT
+
```

Comment:
Add authorship info -- it's something that was added to the other components just now.